lib/pos_mempool.go

```diff
@@ -151,6 +151,8 @@ type PosMempool struct {
 	// based off the current state of the mempool and the most n recent blocks.
 	feeEstimator *PoSFeeEstimator
 
+	maxValidationViewConnects uint64
```

Comment:
Please add a comment above this that explains what `maxValidationViewConnects` does.